docs/faqs.md

```diff
@@ -0,0 +1,43 @@
+# FAQs
+
+## What is a state channel?
+
+A state channel can be thought of as an account with multiple balances (commonly just two). The owners of that account can update those balances according to some rules which they agree on beforehand and which can be enforced on a blockchain.
+
+State channels therefore allow for peer-to-peer games, payments and other few-user applications to safely trade blockchain assets at extremely low latency, low cost and high throughput without requiring trust in a third-party.
+
+## What kind of applications are there?
+
+State channels can be programmed such that assets are redistributed according to arbitary logic, allowing for applications such as poker, conditional payments, atomic swaps and more.
+
+## What is Nitro Protocol?
+
+Nitro protocol is a state of the art state channel protocol which is focussed on security, performance and extensibility. It has been developed over several [years of research](./research-papers.md).
+
+One of the key features of Nitro are [_virtual_ channels](./protocol-tutorial/0060-funding-a-channel.md#fund-virtually), where peers can setup a direct connection with each other entirely off-chain.
+
+## How is it implemented?
+
+The on-chain components of Nitro protocol are implemented in solidity, and are published alongside lightweight off-chain support in Typescript in the npm package [@statechannels/nitro-protocol](https://www.npmjs.com/package/@statechannels/nitro-protocol).
+
+The off-chain component of the protocol are implemented in [`go-nitro`](https://pkg.go.dev/github.com/statechannels/go-nitro), a reference client for Nitro Protocol.
```

Comment:
It might be worth breaking this into a "What is go-nitro" question?

Reply:
Yep I think that's a good shout.  I started out on this but ended up in a bit of rabbit hole thinking about the go-`nitro.Client` API. I'm going to tackle it in a new PR since there are many questions about it. #936